assets/js/googlesitekit/datastore/site/consent-mode.js

```diff
@@ -171,26 +172,25 @@ const baseSelectors = {
 	 * Returns true if Google Ads is in use, either through a linked Analytics & Ads
 	 * account, an Ads conversion tracking ID, or via Analytics tag config.
 	 *
-	 * TODO: The Ads conversion tracking ID is being moved to the new "Ads" module. Source
-	 * this ID from the "Ads" module once it's implemented.
-	 *
 	 * @since 1.124.0
 	 * @since n.e.x.t Updated to consider Ads connection status via the Analytics tag config.
+	 * @since n.e.x.t Updated to source Ads conversion ID field from Ads module.
```

Comment:
This is a minor one, I'd suggest rolling this comment into the previous one so we don't end up with two `@since` tags next to each other for the same version.
```suggestion
	 * @since n.e.x.t Updated to consider Ads connection status via the Analytics tag config, and to source the conversion ID field from Ads module.
```

Reply:
Yeah I was divided between merging it or adding it separately, though it might be too long for a single comment. But I will merge it

Reply:
Done